qa/L0_inferentia_perf_analyzer/test.sh

```diff
@@ -26,7 +26,16 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # First need to set up enviroment
-source ${TRITON_PATH}/python_backend/inferentia/scripts/setup.sh -p
```

Comment:
You should also report appropriate error when both USE_TENSORFLOW and USE_PYTORCH are set 1.